cmd/backup/main.go

```diff
@@ -541,6 +571,7 @@ func (s *script) copyBackup() error {
 // backups, it does nothing instead.
 func (s *script) pruneOldBackups() error {
 	if s.c.BackupRetentionDays < 0 {
+		s.logger.Infof("Won't prune any backups if 'BACKUP_RETENTION_DAYS' isn't set / is lower than 0. (BACKUP_RETENTION_DAYS=%d)", s.c.BackupRetentionDays)
```

Comment:
Considering `0` is the default, I probably wouldn't log this as it could be confusing for people that just use the default configuration and now see messages about how someone is potentially trying to erase their backups again (which would scare me personally at least).

Reply:
Oh yeah! -- This was just for debugging. :D
Deleted in: e18ec1f8989378d28be11e4096f1cd5413a862bb